main/src/cgeo/geocaching/apps/AbstractLocusApp.java

```diff
@@ -352,13 +350,12 @@ private static String toLocusWaypoint(final WaypointType wt) {
             case FINAL:
                 return GeocachingWaypoint.CACHE_WAYPOINT_TYPE_FINAL;
             case OWN:
+            case STAGE:
                 return GeocachingWaypoint.CACHE_WAYPOINT_TYPE_PHYSICAL_STAGE;
             case PARKING:
                 return GeocachingWaypoint.CACHE_WAYPOINT_TYPE_PARKING;
             case PUZZLE:
```

Comment:
I would let it as is.
The code is for me clearer then, the case values are sorted alphabeticly.